pkg/packagekit/updates.jsx

```diff
@@ -604,7 +927,18 @@ class OsUpdates extends React.Component {
         });
     }
 
+    onValueChanged(key, value) {
+        this.setState({ [key]: value });
+    }
+
     componentDidMount() {
+        cockpit.spawn(['which', 'tracer'], { err: 'ignore' })
```

Comment:
The `which` package may not be installed; use `type` instead. However, please consider a more pythonic [EAFP](https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/) approach: Just call tracer, and if you get a "no such file or directory" error, set tracerAvailable to false.

Moreover, we don't actually call the tracer binary any more, but import the Python API instead. That may also be installed through e.g. pip. So, intercept the ImportError either in `callTracer()` or in the callTracer.py, and have that return a "not-available" string or something such.

Reply:
Is there any reason why it's needed to intercept ImportError? If import error happens, and I just let bubble up to updates.jsx, and catch it there (as it's done now), it would not really make a difference, no?

Reply:
Sure, if it already handles a failing python call, that's fine of course.